website/docs/developer/development-guide/code/integration-tests.md

```diff
@@ -122,30 +131,41 @@ This is a list of tests that should be created for existing Orca architecture. T
 - [Error](#error-paths):
   1. Requests for catalog info on a non-existent granule/file should return HTTP Status 404.
 
+  1. Attempting to ingest a file that does not exist should result in `files does not exist`.
+
 ### Recovery
 - [Happy](#happy-path):
   1. Ingest randomized data to Orca.
      :::tip
-     Include a 200 GB file to make sure timeouts do not prevent recovery.
+     Include a large (bigger than 250 Gb) file to make sure timeouts do not prevent recovery.
      :::
   1. Call the OrcaRecoveryWorkflow to restore the files from Orca to another bucket.
+     :::tip
+     Make sure to cover ExcludeFileTypes being set, being unset, and excluding files. May require additional tests.
```

Comment:
May want to expound on this on what the expected result/verification is so that there is no confusion about the behavior, especially when we pass a file that will be filtered.